root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -8,23 +12,90 @@ function OCC
     TERM=dumb runuser -s /bin/bash apache -c "php -dmemory_limit=512M ./occ $params"
 }
 
+exitOnError () {
```

Comment:
When this function is executed, we should get rid also of `tmp_sql`.
Maybe a trap function is better?

Reply:
done